test/onemkl.jl

```diff
@@ -0,0 +1,27 @@
+using oneAPI
```

Comment:
No need to repeat most of the test preamble here, as `onemkl.jl` is included in `setup.jl` from `runtests.jl`. To run in isolation, you can do `julia -L test/setup.jl test/onemkl.jl`